projects/example/src/app/app.module.ts

```diff
@@ -22,22 +22,23 @@ const palette: any = {
     CommonModule,
     BrowserModule,
     AppRoutingModule,
-    FronteggAppModule.forRoot({
-      version: 'next',
-      contextOptions: {
-        baseUrl: 'https://max.frontegg.com'
-      },
-      metadata: {
-        navigation: {
-          users: {
-            visibility: 'hidden',
-          },
+    FronteggAppModule.forRoot(
+      {
```

Comment:
we need to provide another solution to dynamic configuration, so the vendor can initialize our module with his ConfigService